src/odemis/util/polar.py

```diff
@@ -341,24 +427,107 @@ def _CreateMirrorMask(data, pixel_size, pole_pos, hole=True):
     X, Y = data.shape
     pole_x, pole_y = pole_pos
 
+    # create mirror mask in cartesian coordinates
```

Comment:
That's pretty what the whole function does, doesn't it? If so, just drop the comment (or merge it into the docstring, if it wasn't clear).